samples/EverythingServer/Tools/AnnotatedMessageTool.cs

```diff
@@ -41,7 +41,7 @@ public static IEnumerable<ContentBlock> AnnotatedMessage(MessageType messageType
         {
             contents.Add(new ImageContentBlock
             {
-                Data = TinyImageTool.MCP_TINY_IMAGE.Split(",").Last(),
+                Data = System.Text.Encoding.UTF8.GetBytes(TinyImageTool.MCP_TINY_IMAGE.Split(",").Last()),
```

Comment:
nit: consider changing this to a span literal to avoid the need to parse a static resource.  Both DataContent and ImageContentBlock can construct from the raw bytes.